.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,56 @@
+name: Run PHPCS
+
+on: [push]
+
+jobs:
+  test:
+    name: WP ${{ matrix.wordpress }} on PHP ${{ matrix.php }}
+    runs-on: ubuntu-16.04
+    continue-on-error: ${{ matrix.allowed_failure }}
+    
+    env:
+      WP_VERSION: ${{ matrix.wordpress }}
+      
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+        wordpress: [ '5.5', '5.6', '5.7'  ]
+        allowed_failure: [ false ]
+        include:
+          # PHP nightly.
+          - php: '8.1'
```

Comment:
It might be good to exclude the nightly for now as our dependencies aren't compatible with it else we just risk seeing consistent failures (even though they are allowed).

Reply:
Sounds good. I just removed the nightly PHP version.